app/src/main/java/com/haroldadmin/crashyapp/MainActivity.kt

```diff
@@ -2,18 +2,20 @@ package com.haroldadmin.crashyapp
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
-import kotlinx.android.synthetic.main.activity_main.*
+import com.haroldadmin.crashyapp.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        val activityMainBinding = ActivityMainBinding.inflate(layoutInflater)
```

Comment:
A good convention to follow for binding variables is to just call them `binding`. Their context is evident from their parent Activity/Fragment class, so there's no need to include their name in the variable.

Reply:
done